js-extensions/src/ExtensionRenderer.jsx

```diff
@@ -20,8 +20,9 @@ export default class ExtensionRenderer extends React.Component {
     }
 
     componentDidMount() {
-        ExtensionRenderer.resourceLoadTracker.onMount(this.props.extensionPoint);
-        this._renderAllExtensions();
+        ExtensionRenderer.resourceLoadTracker.onMount(this.props.extensionPoint, function() {
+            this._renderAllExtensions();
+        });
```

Comment:
So now, we only call `this._renderAllExtensions()` after all CSS has ben loaded.